prompts/demo_flow.ipynb

```diff
@@ -20,6 +20,7 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    from utils import SQLConnector
</pre><p>can you modify the SQLConnector in <code>utils</code> to add a 4 second time delay when using SQLite? Otherwise the SQL spans appear too small in the Trace timeline in W&amp;B and its hard to see them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/edu/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>